homeassistant/components/mqtt/light/schema_json.py

```diff
@@ -154,13 +242,23 @@ def valid_color_configuration(config: ConfigType) -> ConfigType:
 )
 
 DISCOVERY_SCHEMA_JSON = vol.All(
+    valid_color_configuration(False),
     _PLATFORM_SCHEMA_BASE.extend({}, extra=vol.REMOVE_EXTRA),
-    valid_color_configuration,
+    cv.deprecated(CONF_COLOR_MODE),
```

Comment:
`valid_color_configuration` logs + raises issue, don't we get double logging if also using `cv.deprecated`?

Reply:
You are right, we should remove the `cv.deprecation` logging.